src/components/input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import { TextField } from "@mui/material";
+import React from "react";
+
+
+interface InputProps {
+  type: string;
+  placeholder: string;
+}
+
+
+const Input: React.FC<InputProps> = (props) => {
```

Comment:
Your are limiting the TextField API to only two props while you can use the interface provided by materail-ui for the text field